windows_containers/enabling-windows-container-workloads.adoc

```diff
@@ -33,7 +33,7 @@ Windows instances deployed by the WMCO are configured with the containerd contai
 
 [role="_additional-resources"]
 .Additional resources
-* For the comprehensive prerequisites for the Windows Machine Config Operator, see xref:../windows_containers/understanding-windows-container-workloads.adoc#wmco-prerequisites_understanding-windows-container-workloads[Understanding Windows container workloads].
+* For the comprehensive prerequisites for the Windows Machine Config Operator, see xref:../windows_containers/windows-containers-release-notes-6-x#wmco-prerequisites_windows-containers-release-notes[Understanding Windows container workloads].
```

Comment:
I think the link text here should change to indicate that the link points to release notes instead of "Understanding Windows container workloads".